### PR TITLE
[MRG] CTA split for prospects and clients

### PR DESCRIPTION
**Pull request by michelfernandez**
This PR adds separate CTA for contacting sales directly in the header. Due to this change, the navigation has been slightly adapted. A secondary corresponding CTA has also been added to the intro and outro components.

### CODE REVIEW

File: site/data/webpack.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "js": "main.js"
   },
   "": {
-    "": "fonts/.keep",
+    "keep": "fonts/.keep",
```
Comment (fursinger) [inline site/data/webpack.json:7]:
```suggestion
    "": "fonts/.keep",
```

**Comment by dkumpf1** [comment]
[DEPLOY-PREVIEW]

**Tested: OK**

I tested this page on all browsers (Chrome/FF/Chromium) and on mobile (Native/Chrome) with every language (DE/EN/FR) and it looks ok.

**Comment by fursinger** [comment]
I removed the tested: OK label because changes have been made after setting the label.

**Comment by dkumpf1** [comment]
[DEPLOY-PREVIEW]

**Tested: OK**

I tested all the changes that are mentioned above and the overall pages in all browers (Chrome/FF/Chromium) together with mobile (Chrome/Native) on all languages (DE/EN/FR). I think the changes look good from my point of view.

**Comment by fursinger** [comment]
Kira gave ok her ok to merge.

**Pull request merged**